spec/models/email_link_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe EmailLink, type: :model do
+    it "default factory is valid" do
+       expect(build(:email_link).valid?).to be_truthy
+
```

Comment:
some extra space here between the `expect` and the `end` line. Please have at most one blank line.